Interop/ModelCard/src/main/java/org/tribuo/interop/modelcard/package-info.java

```diff
@@ -15,5 +15,81 @@
  */
 /**
  * ModelCard feature to allow more transparent model reporting.
+ * Refer to <a href="https://arxiv.org/abs/1810.03993">Model Cards for Model Reporting</a> to read the original paper.
+ * <p>
+ * Using the ModelCard package, users can construct a {@link org.tribuo.interop.modelcard.ModelCard} object that partially documents
+ * a trained model using Tribuo's built-in provenance. The remainder of the documentation
+ * (i.e., adding extracted testing metrics and specifying the model's {@link org.tribuo.interop.modelcard.UsageDetails})
+ * can either be left as blank, added by the user programmatically, or added by the user via
+ * {@link org.tribuo.interop.modelcard.ModelCardCLI}.
+ * <p>
+ * A serialized ModelCard object with empty strings and lists set as the fields of {@link org.tribuo.interop.modelcard.UsageDetails} may
+ * look like the following:
+ * <p>
+ * {
```

Comment:
Can you surround this with `<pre>` tags? Otherwise it'll render it as plain text and reformat it into a paragraph.